<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortControlWindowClass</class>
 <widget class="QMainWindow" name="SerialPortControlWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortControlWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="StopBitslabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="Baudratelabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataBitslabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检测串口</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="BaudrateBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DataBitsBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="PIDCheckBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电流环控制</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>阶跃反馈力矩</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="TxdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>0.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Txlabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Tx</string>
      </property>
     </widget>
     <widget class="QLabel" name="Tylabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Ty</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="TydoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>0.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QSlider" name="TxSlider">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-50</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="TySlider">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-50</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>正弦反馈力矩</string>
     </attribute>
     <widget class="QLabel" name="Wylabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Wy</string>
      </property>
     </widget>
     <widget class="QLabel" name="Wxlabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Wx</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="WydoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>6.300000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="WxdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>6.300000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Axlabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Ax</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="AydoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>0.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="AxdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>0.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="Aylabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Ay</string>
      </property>
     </widget>
     <widget class="QSlider" name="TxSinSlider">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-500</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="TySinSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>22</width>
        <height>160</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-500</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Paritylabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PortNameBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PortNamelabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ParityBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="GravityCheckBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>重力补偿</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StopBitsBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ZeroButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>力矩清零</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SerialPortControlWindow.qrc"/>
 </resources>
 <connections/>
</ui>
